packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1488,6 +1683,121 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
     _currentController = null;
   }
 
+  /// Handler called when a long press gesture has started on the leading
+  /// or trailing scrollbar's buttons.
+  @protected
+  @mustCallSuper
+  void handleButtonsPressDown(
+      Offset localPosition, {
+        required bool isLeadingButton,
+        Duration duration = const Duration(milliseconds: 100),
+      }
+      ) {
```

Comment:
The formatting looks a bit off here. 
I wonder, would it be beneficial to create a new gesture recognizer like this? I can imagine more use cases than jus the scrollbar, a press and hold callback repeater. I also would not be surprised if after adding this users wanted more control over configuring it.